src/main/java/kotori/Kotori.java

```diff
@@ -0,0 +1,88 @@
+package kotori;
+
+import java.util.Scanner;
+
+import kotori.command.Command;
+import kotori.command.ExitCommand;
+import kotori.command.GreetCommand;
+import kotori.parser.Parser;
+import kotori.storage.Storage;
+import kotori.taskList.TaskList;
+
+
+
+/**
+ * This class represents the bot.
+ * */
+
+public class Kotori {
+    private Storage storage;
+    private TaskList taskList;
+    private Parser parser;
+
+    /**
+     * Produce a bot.
```

Comment:
Verbs should be of the form XXXs, in this case, it should be "Produces" instead of "Produce"